requirements-dev.txt

```diff
@@ -351,7 +351,7 @@ pytest-rerunfailures==14.0
     # via localstack-core
 pytest-split==0.9.0
     # via localstack-core
-pytest-tinybird==0.2.0
+pytest-tinybird==0.3.0
```

Comment:
https://github.com/tinybirdco/pytest-tinybird/releases/tag/v0.3.0

@alexrashed useful to know if we run into some tinybird timeout issues in the future